sources/packages/web/src/services/FormUploadService.ts

```diff
@@ -46,18 +47,13 @@ export default class FormUploadService {
 
   public async downloadFile(fileInfo: FormUploadFileInfo) {
     try {
-      const fileContent = await ApiClient.FileUpload.download(fileInfo.url);
-      // Change the storage type to base64 to allow the file to be "downloaded"
-      // using the bytes retrieved instead of just opening an url.
-      // if we use directly the url we will not have the oportunity to authorize
-      // the file download.
-      return {
-        storage: "base64",
-        url: fileContent,
-        originalName: fileInfo.originalName,
-        name: fileInfo.name,
-        type: fileInfo.type,
+      const fileUtils = useFileUtils();
+      const studentDocumentDetails = {
+        fileName: fileInfo.originalName,
+        uniqueFileName: fileInfo.name,
+        fileOrigin: FileOriginType.Temporary,
```

Comment:
Because of the reason that input model of  `downloadStudentDocument` is expecting  fileOrigin we are manualling setting the value to `FileOriginType.Temporary`. I would say update the type of `downloadStudentDocument` input to expect the properties required.

Reply:
Something like this. 
![image](https://github.com/user-attachments/assets/8fba41c9-8fe7-4c69-8f44-7cb6faf0df7d)